frontend/components/common/Layout/Footer.tsx

```diff
@@ -290,27 +296,37 @@ function NewsletterForm({
                   ref={inputRef}
                   disabled={subscription.status !== SubscriptionStatus.Idle}
                   placeholder={emailPlaceholder}
+                  hidden={subscription.status === SubscriptionStatus.Subscribed}
                />
+               {subscription.status === SubscriptionStatus.Subscribed && (
+                  <Text align="center">
+                     <Icon as={RiCheckFill} boxSize="5" mb="-5px" />
+                     Subscribed!
+                  </Text>
+               )}
```

Comment:
I think this could be simpler if you absolutely position this rather than trying to put it above a zero-height thing in a column.

Reply:
Resolved b1dc8261947d1bb18530a49913791ffe82cbb4ab